app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss

```diff
@@ -1,54 +1,67 @@
 @import '~styles/settings.scss';
 
 .c-widget-settings {
-  width: rem(250px);
+  width: rem(310px);
   background-color: $white;
   padding: rem(25px) rem(25px) rem(10px);
 
   > div {
     margin-bottom: rem(20px);
-    border-bottom: solid 1px $border;
 
     &:last-child {
-      border: 0;
+      margin-bottom: 0;
```

Comment:
This breaks some cases
![screen shot 2018-06-14 at 12 12 42](https://user-images.githubusercontent.com/20288774/41406291-424d2d90-6fcc-11e8-8475-a52f566ad640.png)